docs/components/light.md

```diff
@@ -50,6 +50,14 @@ The example below creates a light source shining from the upper-left at a 45-deg
 <a-entity light="type: directional; color: #EEE; intensity: 0.5" position="-1 1 0"></a-entity>
 ```
 
+It is also possible to specify the direction of the directional light with it's orientation by creating a child entity it targets. for example pointing down it's -Z axis:
```

Comment:
the nested target needs a position offset to set what relative position from the light object3D to point at. if it was 0 0 0 it wouldn't make sense, pointing from 000 to 000... So setting position of the target to 0 0 -1 sets it to target along it's Z axis.

last thing i am wondering about concerning this light target is if we should handle spotlights the same as above, ie with a nested target to allow for rotation instead of what is now comitted with spots having the nested target by default. This will break scenes that have a positioned spotlight that depended on the target to be at world 0 0 0. So either make that clear in the change notes... That to keep the old behaviour one must create a not nested target entity at 0 0 0. Or i need to commit a fix to do the same like directional lights, and we add the info how to do rotation to the spotlight docs too.